src/test/resources/it/pagopa/pn/cucumber/workflowNotifica/webhook/pf/AvanzamentoNotificheWebhookB2bPF.feature

```diff
@@ -180,9 +180,9 @@ Feature: avanzamento notifiche webhook b2b
     And destinatario Mario Cucumber e:
       | digitalDomicile_address | test@fail.it |
     And si predispone 1 nuovo stream denominato "stream-test" con eventType "TIMELINE"
-    And si crea il nuovo stream per il "Comune_1"
-    When la notifica viene inviata tramite api b2b dal "Comune_1" e si attende che lo stato diventi ACCEPTED
-    Then vengono letti gli eventi dello stream del "Comune_1" fino all'elemento di timeline "NOT_HANDLED"
+    And si crea il nuovo stream per il "Comune_Multi"
```

Comment:
ATTENZIONE!!!!!
I Feature file del webhook sono "divisi per comune" (nel senso che in ogni file viene usato un SOLO COMUNE) per evitare problemi di concorrenza

Reply:
fatto